src/main/kotlin/no/nav/syfo/aktivitetskrav/domain/AktivitetskravVurdering.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.syfo.aktivitetskrav.domain
+
+import no.nav.syfo.domain.PersonIdent
+import java.time.OffsetDateTime
+import java.util.*
+
+data class AktivitetskravVurdering(
+    val uuid: UUID,
+    val personIdent: PersonIdent,
+    val createdAt: OffsetDateTime,
+    val status: AktivitetskravStatus,
+    val vurdertAv: String,
+    val sistVurdert: OffsetDateTime,
+) {
+    fun isFinalVurdering() = status in finalVurderinger
```

Comment:
"Final" er bare noe jeg fant på, vi kan godt snakke om hva vi kaller slike "finaltilstander" eller "slutttilstander" for aktivitetskravet. `Utfall` er vel også noe TH har brukt om de ulike 🤔